christfellowshipchurch/web-app
# Add viewed interaction to live stream actions

# To Test

- Make sure you pull on the api. A new enum was added called `VIEWED_ACTION`
- Go to a live stream event and click on one of the actions. If you have graphQL devtools you can check to see if the `interactWithNode` mutation ran successfully. And check Rock interactions to see if it was recorded there too. Most likely you will see it here if you are testing the `Test Event - 24 Hour Live Stream` https://rock.gocf.org/page/664?ComponentId=181648

# Screenshots
![image](https://user-images.githubusercontent.com/2528817/98870974-bc8c9e00-2439-11eb-9fae-4d70cf5baeb7.png)

![image](https://user-images.githubusercontent.com/2528817/98870856-83ecc480-2439-11eb-9d95-686ab1e8a964.png)

![image](https://user-images.githubusercontent.com/2528817/98999844-05a62600-24fe-11eb-93a5-449206375742.png)

# TODO
- [x] verify Rock is recording interaction
